src/tests/e2e/pages/accounts.page.js

```diff
@@ -0,0 +1,48 @@
+import log from 'loglevel'
+log.setDefaultLevel('info')
+
+class AccountsManagementPage {
+  async getAccountsList () {
+    const accountsElementList = await page.$$('[data-test-id^="account_list_item"]')
+    const accounts = await Promise.all(
+      accountsElementList.map(async elementHandle => {
+        const walletPlatformHandle = await elementHandle.$('[data-test-id="wallet_platform"]')
+        const assetsHandle = await elementHandle.$('[data-test-id="assets_cell"]')
+        const walletType = (
+          await (await walletPlatformHandle.getProperty('textContent')).jsonValue()
+        ).split(', ')[0]
+        const platformType = (
+          await (await walletPlatformHandle.getProperty('textContent')).jsonValue()
+        ).split(', ')[1]
+        let assets = await (await assetsHandle.getProperty('textContent')).jsonValue()
+        while (!assets) {
```

Comment:
is it possible to simply wait till skeleton does not show up anymore

Reply:
It's basically the same thing

Reply:
we can do something like this to better control the timeout. Currently approach depends on test timeout
await page.waitForSelector('div', {hidden: true, timeout: xxx});

Reply:
but we dont know how long it might take

Reply:
good point